fapolicy_analyzer/ui/ancillary_trust_database_admin.py

```diff
@@ -130,11 +137,31 @@ def on_files_added(self, files):
 
     def on_trustBtn_clicked(self, *args):
         if self.selectedFiles:
-            self.add_trusted_files(*self.selectedFiles)
+            dne_list = [f for f in self.selectedFiles if not os.path.isfile(f)]
+            selectedFiles = [f for f in self.selectedFiles if f not in dne_list]
+            if len(dne_list) > 0:
+                removeDialog = RemoveDeletedDialog(deleted=dne_list).get_ref()
+                resp = removeDialog.run()
+                removeDialog.destroy()
+                if resp == Gtk.ResponseType.APPLY:
+                    self.delete_trusted_files(*dne_list)
```

Comment:
This should never happen right? Being able to click the Trust button on a file that's missing.